docs/guides/save_tf_state_in_vkcs_s3.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: "vkcs"
+page_title: "Save Terraform state in VKCS S3"
+description: |-
+  Save Terraform state in VKCS S3
+---
+
+## Configure a bucket
+
+You will need a bucket to save the terraform state, you can do this by following one of these documentations: 
+1) [Create bucket using terraform](https://cloud.vk.com/docs/en/tools-for-using-services/terraform/how-to-guides/aws)
+2) [Create bucket using website or aws cli](https://cloud.vk.com/docs/en/storage/s3/service-management/buckets/create-bucket)
+
```

Comment:
need links to
1 create credentials
2 choose region and endpoint

Reply:
and we'll need a warning how to store credentials securely (we have similar warnings somewhere in the code)

Reply:
done